javaparser-core/src/main/java/com/github/javaparser/resolution/declarations/AccessLevel.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 Federico Tomassetti
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.javaparser.resolution.declarations;
+
+/**
+ * @author Federico Tomassetti
+ */
+public enum AccessLevel {
```

Comment:
Didn't we have this one already?

Reply:
Probably, as a first step I just imported stuff from JSS, renamed things and made it compile, now it is time for polishing and removing duplication

Reply:
Just removed and renamed the related interface `HasAccessLevel` to `HasAccessSpecifier`